src/index.js

```diff
@@ -0,0 +1,122 @@
+const urlLocation = 'http://127.0.0.1:5000/location';
+const urlWeather = 'http://127.0.0.1:5000/weather';
```

Comment:
Nice use of global constants! Consider naming them ALL_CAPS to it clear that they're constant configuration to be used globally.